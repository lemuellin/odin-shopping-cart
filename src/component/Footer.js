import '../style/footer.css';
import github from '../asset/contact/github.png';

const Footer = () => {
    return(
        <footer>
            <h5>Copyright Â© 2022 lemuellin</h5>
            <a href='https://github.com/lemuellin' target="_blank" rel="noopener noreferrer">
                <img src={github} alt='github' className='iconFooter'/>
            </a>
        </footer>
    )
}

export default Footer;