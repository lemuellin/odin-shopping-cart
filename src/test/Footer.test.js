import React from "react";
import { render } from "@testing-library/react";
import Footer from '../component/Footer';

describe("Footer", () => {
    it("Match Snapshot", () => {
        const { container } = render (<Footer/>);
        expect(container).toMatchSnapshot();
    });

});