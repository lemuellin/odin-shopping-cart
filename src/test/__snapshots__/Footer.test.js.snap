// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Match Snapshot 1`] = `
<div>
  <footer>
    <h5>
      Copyright Â© 2022 lemuellin
    </h5>
    <a
      href="https://github.com/lemuellin"
      rel="noopener noreferrer"
      target="_blank"
    >
      <img
        alt="github"
        class="iconFooter"
        src="github.png"
      />
    </a>
  </footer>
</div>
`;
